.github/workflows/bump-cli.yml

```diff
@@ -0,0 +1,44 @@
+name: Bump CLI version
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
+  - cron: 0 0 */14 * * # run every 14 days
```

Comment:
Yeah it's a no-op if the version hasn't changed. I had the `workflow_dispatch` as trigger but then I was confused since I thought it may make things weird because I wasn't sure how we wanted this to be executed - I also thought of creating chats-ops but then went against that idea since that would probably be something long term. I assumed a lot of things when I tried experimenting 😂

Reply:
Manual triggers via the actions page is probably fine. This is how we trigger many of our internal release workflows.